experimental-examples/kitchen-sink/tina/custom-component-reference-select/reference-field-schema.tsx

```diff
@@ -1,28 +1,53 @@
 import React from 'react'
 import AuthorCollectionCustomReference from '../../component/custom-reference-select-author'
 import PostCollectionCustomReference from '../../component/custom-reference-select-post'
-import { CollectionProps, COLLECTIONS, InternalSys } from './model'
+import { type CollectionProps, COLLECTIONS, type InternalSys } from './model'
+import type { ReferenceField } from '../../../../packages/@tinacms/schema-tools/src'
```

Comment:
should this be referencing the code directly? or making a package reference?

Reply:
that is a very good point.... let me double check that one

Reply:
should be a packages reference , so we cant use the ReferenceField  from schema tools here

Reply:
This seems to be the case - removed for the moment as this type doesn't seem to be exported... I feel like it should be though...